torch/__init__.py

```diff
@@ -1586,6 +1589,37 @@ def __call__(self, model_, inputs_):
         return self.compiler_fn(model_, inputs_, **self.kwargs)
 
 
+################################################################################
+# Exposing torch.export() and related support APIs
+################################################################################
+from ._constraint import Constraint
+
+def export(
+    f: Callable,
+    args: Tuple[Any],
+    kwargs: Optional[Dict[str, Any]] = None,
+    constraints: Optional[List[Constraint]] = None,
+) -> "torch.export_utils.ExportedProgram":
+    """
+    Traces either an nn.Module's forward function or just a callable with PyTorch
```

Comment:
Does this mean that this needs to be called as `torch.export(mod.forward, ...)` ?